src/Render/Program.cs

```diff
@@ -52,17 +54,25 @@ internal static class Program {
 
 			var renderer = TemplateRenderer.CreateFromResource( "Templates.page.html" );
 			var formatted = await renderer.RenderAsync(
-				title: "TODO: Get Title",
+				title:GetTitle( doc ),
 				content: html
 			);
 
 			outputHtml.Write( formatted );
 		}
 
+		private static string GetTitle( MarkdownDocument doc ) {
+			var inline = doc.Descendants<HeadingBlock>().FirstOrDefault(
+				h => h.Level == 1
+			)?.Inline;

```

Comment:
```suggestion
			var inline = doc.Descendants<HeadingBlock>().Single(
				h => h.Level == 1
			).Inline;
```

There should be exactly one H1 (see #21 about enforcement) so it's OK to crash here.

(I'm imagining that analyzers would run prior to rendering).